### PR TITLE
VATNZ Pacific Positions Update

### PR DESCRIPTION
**Pull request by dirtyformal**
Hey there,

VATNZ has started a Pacific Refresh project, aiming to modernise and overhaul our Pacific positions. As a part of this, we have chosen to do the following -

- Reformat **NSFA_CTR** from a non-standard FIR into a Enroute position. This covers both Western and Eastern Samoa.
- Create **NTTC_CTR** (Tahiti Control). This is an Enroute position in the NTTT FIR that provides both a Class D/E service around the island of Tahiti/Moorea/Bora.
- Create **NCRG_CTR** (Rarotonga Control). This is an Enroute position in the NZZO FIR that provides a Class D Enroute service to the Cook Islands.

These are not sub-sector splits - only new Enroute positions that do not have any existing coverage. I have also provisionally discussed this PR with Niels over Discord a handful of days ago.

Happy to answer any questions!

### CODE REVIEW

File: VATSpy.dat [hunk 1]
```diff
@@ -18083,11 +18089,13 @@ MTEG|Port-Au-Prince||MTEG
...
 MYNN|Nassau|MYNN|MYNN
 NAT|Shanwick & Gander|NAT|NAT
+NCRG|Rarotonga|NCRG
 NFFF|Nadi Oceanic||NFFF
 NFFN|Nadi||NFFN
 NVVV|Port Vila||NVVV
 NSTU|Pago Pago||NZZO
-NTTT|Tahiti Radio||NTTT
+NTTT|Tahiti||NTTT
+NTTC|Tahiti||NTTC
```
Comment (Adrian2k) [inline VATSpy.dat:18098]: These will be called "Tahiti Center" on VATSpy as you have not defined the callsign ending on line 120:

French Polynesia|NT|

Make it 
French Polynesia|NT|Control

File: VATSpy.dat [hunk 2]
```diff
@@ -18083,11 +18089,13 @@ MTEG|Port-Au-Prince||MTEG
 MUFH|Havana|HAV|MUFH
 MYNN|Nassau|MYNN|MYNN
 NAT|Shanwick & Gander|NAT|NAT
+NCRG|Rarotonga|NCRG
```
Comment (Adrian2k) [inline VATSpy.dat:18092]: Same as on NTTT but line 92 - Cook Islands|NC|

File: VATSpy.dat [hunk 3]
```diff
@@ -9728,12 +9731,14 @@ NZWB|Woodbourne (Mil/Civ)|-41.518333|173.870278||NZCT|0
...
 NZWF|Wanaka|-44.738718|169.237232||NZCS|0
+NZWG|Whangamata|-37.200556|175.866667||NZAA|0
+NZWH|Wellington Hospital|-41.308333|174.779444||NZCT|0
 NZWJ|Wellsford|-36.284694|174.522778||NZAA|0
 NZWK|Whakatane|-37.922546|176.920215|WHK|NZCB|0
 NZWL|West Melton|-43.474615|172.388591||NZCS|0
 NZWM|Waimate|-44.789092|171.075519||NZCS|0
-NZWN|Wellington|-41.335264|174.803487|WLG|NZCT|0
 NZWN|Wellington|-41.335264|174.803487|NZWN-F|NZCT|1
+NZWN|Wellington|-41.335264|174.803487|WLG|NZCT|0
```
Comment (Adrian2k) [inline VATSpy.dat:9741]: By the look of it, WLG is not used as a logon for this airport, atleast I can not find it being used. However NZWN_DEL/GND/TWR/APP is being used.

If this is correct, then we should only have one entry for the airport, being `NZWN|Wellington|-41.335264|174.803487|NZWN-F|NZCT|0`

**Comment by dirtyformal** [comment]
@Adrian2k - Thanks for taking a look over it! You're quite right - those positions are meant to have the "Control" terminology over the "Centre" term. Thank you for that!

I've also amended the WLG error.

**Review by NelisV — CHANGES_REQUESTED** [review]
image
Ah, sorry i'm a bit late with reviewing this, but you need to move these polygons to the other side of the antimeridian (east of -180).

**Review by Adrian2k — APPROVED** [review]
The VATSpy.dat file is good now.
Firboundaries needs to be resolved as Niels noted, so he will add the final approveal once its ready.

**Comment by dirtyformal** [comment]
> Ah, sorry i'm a bit late with reviewing this, but you need to move these polygons to the other side of the antimeridian (east of -180).

Sorry it took me a hot minute to figure out how damn maps work, but I've found and fixed a problem on our end which should rectify the meridian issues.

Thanks for your patience with this - I appreciate the help from both of you!

**Review by NelisV — APPROVED** [review]

**Comment by NelisV** [comment]
Ah no problem, thanks for taking the effort. All looking good now ;)

**Review by NelisV — CHANGES_REQUESTED** [review]
So we do have a conflict, i think it's because you based this on the master branch instead of 2204. I think it's best to move this to 2205 as you already suggested.

**Comment by dirtyformal** [comment]
> So we do have a conflict, i think it's because you based this on the master branch instead of 2204. I think it's best to move this to 2205 as you already suggested.

My apologies for the error - I'll give it a look when I get home. I may even close this PR, and open a new one based on 2204.

Would there be any potential for a mid-cycle release - or is that off the table?

**Comment by NelisV** [comment]
> > So we do have a conflict, i think it's because you based this on the master branch instead of 2204. I think it's best to move this to 2205 as you already suggested.
> 
> My apologies for the error - I'll give it a look when I get home. I may even close this PR, and open a new one based on 2204.
> 
> Would there be any potential for a mid-cycle release - or is that off the table?

Nope, we really try to stick to the airac cycles for our own sanity. So 2205 would be the next opportunity.

**Comment by dirtyformal** [comment]
Closing this PR in order to base off the 2205 branch - hopefully solving the conflicts.

**Pull request closed without merging**